### DISCUSSION TITLE
New Forest Route Development

### DISCUSSION THREAD

**Discussion by rickloader** [rickloader/NewForestRouteV3#19] [Announcements]
Development has been hindered by my daughters illness. I can take a laptop to hospital, but I can`t take the large number of reference books needed. 
Work on the route will be mainly  restricted to track expansion on the Woking - Salisbury axis, and some timetable additions.
There will be many minor updates on Github to keep desktop and laptop in synch. I hope users of NF route will understand that while work continues, additions will be somewhat haphazard.
Rick

**Comment by djw-zr** [comment]
Sorry to hear about your daughter.  Concentrate on helping her.  I hope she recovers soon.

Best regards,

David

**Comment by rickloader** [comment]
Thank you David. My daughter faces a long recovery, but is making progress.
The route development will be 1) geo files 2) embankments and bridges 3) line side fences 4) terrtex repaints 5) station platforms 6)general scenery, At the same time a new timetable will extend to Woking with some electric trains.
Inevitably the route size will grow, and my apologies to those who think the size is already  excessive I hope the enlarged system is enjoyable.
The final extent of NewForest Route will be Woking - Weymouth and Woking - Salisbury. Time span? about 2 years
Rick

**Comment by rickloader** [comment]
Route development now extends along the West of England Mainline to Salisbury and Templecombe., with some secondary routes.
The original Bournemouth line is V3 and the expanded route designated V4. A new timetable "V4"  is being created to extend both the Bournemouth and West of England trains over the new tracks. V3 paths are retained for compatibility.
So we have The South Western mainline From Woking to Basingstoke. Here the route splits into the West of England and the Bournemouth lines. Secondary lines connect at Andover and Salisbury ,  and with the Somerset &Dorset at Templecombe. The map also shows how the terrain maps are distributed
 
![NewForestMap_V4](https://github.com/rickloader/NewForestRouteV3/assets/57818551/b9a7c9e1-5b51-4d03-913c-0f5b544d05bf)
